TLSharp.Core/DataCenter.cs

```diff
@@ -1,21 +1,20 @@
-
 namespace TLSharp.Core
 {
     internal class DataCenter
     {
-        internal DataCenter (int? dcId, string address, int port)
+        internal DataCenter(int? dcId, string address, int port)
         {
             DataCenterId = dcId;
             Address = address;
             Port = port;
         }
 
-        internal DataCenter (string address, int port) : this (null, address, port)
+        internal DataCenter(string address, int port) : this(null, address, port)
```

Comment:
same here

Reply:
Fixed it

Reply:
can you squash the 2nd commit with the 4th commit?

Reply:
Done